cameras/paths.csv

```diff
@@ -16,6 +16,7 @@ Axis,nvr,241Q,(00:40:8C),rtsp://{{username}}:{{password}}@{{ip_address}}/mpeg4/{
 Hikvision,camera,DS-2CD2032-I;DS-2CD2432F-IW;DS-2CD2035-I;DS-2CD2042WD-I;DS-2CD2342WD-I;DS-2CD2142FWD;DS-2CD4A26FWD-IZHS/P;DS-2CD4135F-IZ,(C4:2F:90|C0:56:E3|BC:AD:28|44:19:B6|28:57:BE|00:95:69|A4:14:37|20:F4:1B|54:C4:15|4C:BD:8F|B4:A3:82|64:DB:8B|94:E1:AC|18:68:CB|58:03:FB|BC:BA:C2|14:2F:FD|64:25:5E),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/PSIA/streaming/channels/10{{stream}},H.264,554,1;2,,,,2,1,admin,admin,,http://www.hikvision.com/UploadFile/image/2013061411054615328.pdf
 Hikvision,nvr,DS-7600 series,(8C:E7:48|BC:AD:28|18:68:CB),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/Streaming/Channels/{{channel}}{{stream}},H.264,554,01;02,1;2;3;4;5;6;7;8,hi-res;lo-res,Channel {{channel}},02,01,admin,12345,,http://www.hikvision.com//uploadfile/image/20150831025213965.PDF
 Hikvision,nvr,DS-7600 series,(54:C4:15|4C:BD:8F),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/Streaming/Channels/{{channel}}{{stream}},H.264,10554,01;02,1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16,hi-res;lo-res,Channel {{channel}},02,01,admin,12345,,http://www.hikvision.com//uploadfile/image/20150831025213965.PDF
+Hikvision,nvr,NVR,(68:6D:BC),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/Streaming/Channels/101,H.264,554,101;102;201;202,admin,12345,,,
```

Comment:
Is this a duplicate of line 17?
It looks like we should modify only the OUI regex of line 17 instead by adding `|68:6D:BC` to the regex unless this really is a special 2-Chanel only NVR?